src/main/java/org/jenkinsci/plugins/ghprb/extensions/GhprbBuildStep.java

```diff
@@ -6,7 +6,7 @@
 import hudson.model.Action;
 
 public interface GhprbBuildStep extends Action {
-    public static String buildStep = "GhprbBuildStep";
+    String buildStep = "GhprbBuildStep";
```

Comment:
All methods in an interface are public, it's redundant as it can not be anything but public.